docs/jvm_aot.md

```diff
@@ -1,18 +1,20 @@
-# Ahead Of Time (AOT) compilation for the Java Virtual Machine (JVM)"
+# Ahead Of Time (AOT) compilation for the Java Virtual Machine (JVM)
 
 Ahead Of Time (AOT) compilation build tools such as those provided by [GraalVM's `native-image`](https://www.graalvm.org/reference-manual/native-image/) can require some configuration when using protobuf.
-Protobuf for the JVM uses reflection and some of its target classes are not possible to determine in advance.
-Historically, there were good reasons to use reflection based on APIs that were published effectively requiring them, and this situation is unlikely to change.
+Protobuf for the JVM uses reflection, and some of its target classes are not possible to determine in advance.
+This situation is unlikely to change.
```

Comment:
Removing the historical context here gets rid of a breadcrumb that might help someone understand the motivation.

Reply:
Speaking as one data point, I can't follow this breadcrumb.